### PR TITLE
Add a "people of taskcluster" page to the docs

### PR DESCRIPTION
**Pull request by djmitche**
This is generated using public Mozillians API data.

I'm working with the Mozillians folks to find a better way to filter only public data, and I haven't set up the automatic builds yet, but I'll figure that out before I merge.  In the interim, if you want to see the results, you'll need to run it locally with your own Mozilians API key in `$MOZILLIANS_API_KEY`.

Note that a *lot* of the accounts this links to are non-public.  I was thinking of reaching out to the relevant people to let them know (a) you're in the credits and (b) private is fine, but if you want more detail included, consider making your profile public.

### CODE REVIEW

File: lib/userlist.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+var gutil = require('gulp-util');
+var through = require('through2');
+var Mozillians = require('mozillians-client');
+var yaml = require('js-yaml');
+var pug = require('pug');
+
+// Take a file containing a PUG template, along with file.data.people containing
+// a list of mozillians usernames, and produce HTML output.
+module.exports = function (options) {
+	return through.obj(function (file, enc, cb) {
```
Comment (eliperelman) [inline lib/userlist.js]: Spacing here looks really weird.

**Comment by eliperelman** [comment]
Can we make each content box linkable?

**Comment by djmitche** [comment]
I took away the box and used flexbox - what do you think?

**Comment by djmitche** [comment]
screenshot from 2017-03-10 18-06-15

**Comment by djmitche** [comment]
BTW, if we mark things public, like full name, they will show up here.

**Comment by eliperelman** [comment]
:heart_eyes:

**Comment by petemoore** [comment]
@djmitche Thanks for setting this up. I spotted some additional contributors that could also be included (with their permission, of course):

* https://github.com/taskcluster/generic-worker/graphs/contributors
* https://github.com/taskcluster/taskcluster-queue/graphs/contributors

Can people be included that don't have a Mozillians profile?

**Comment by djmitche** [comment]
@petemoore no, mozillians profile is required.  The "Contributor Funnel" describes how people get added to this page. I'm happy to help you navigate that funnel.

**Pull request merged**